spec/services/benefit_check_service_spec.rb

```diff
@@ -30,8 +51,9 @@
       end
 
       it "sends the right parameters" do
-        allow(Savon).to receive(:client).and_return(savon_client)
-        expect(savon_client).to receive(:call).with(:check, expected_params)
+        allow(Faraday::SoapCall).to receive(:new).and_return(faraday)
+        allow(faraday).to receive(:call).with(payload).and_return(response)
+        expect(faraday).to receive(:call).with(payload)
```

Comment:
These two lines are almost identical - is that intentional? Do they both need to be there?

Reply:
@agoldstone93 rubocop seemed to demand the two separate lines (basically it doesn't allow you to specify the and_return with an expect but it also wants the test to have an expect not just an allow).

Reply:
You could try switching it to:
```
it "sends the right parameters" do
        allow(Faraday::SoapCall).to receive(:new).and_return(faraday)
        benefit_check_service.call
        allow(faraday).to have_received(:call).with(payload).and_return(response)
end
```
but it may cause other issues! 🤷‍♂️

Reply:
I'd guess that Rubocop won't be happy with no `expect` in the block @colinbruce

Reply:
Yes, I just tried that and rubocop is not happy with the fact that there are no expectations found in this example.

Reply:
🤦‍♂️ I changed the received verb and forgot to change allow to expect!

Reply:
I'm not seeing these changes. Have you pushed @kmahern?

Reply:
@agoldstone93 apologies, should be on last commit now.